Android/ConstraintLayout/source/ConstraintLayout/ConstraintLayout.csproj

```diff
@@ -44,6 +51,19 @@
     <Reference Include="System.Xml" />
     <Reference Include="System.Core" />
     <Reference Include="Mono.Android" />
+    <Reference Include="Xamarin.Android.Support.Annotations" />

```

Comment:
You shouldn't need to explicitly add references here. The PackageReference below to takes care of all this for you from the top level package reference declarations.